Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM registry.redhat.io/ubi8:latest as builder
+
+# Install required RPMs and ensure that the packages were installed
+USER root
```

Comment:
You don't need to specify root here, the default user for the `ubi8` image is root.

Reply:
ok removed.